frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -303,8 +303,22 @@ class Worksheet extends React.Component {
         // The uuid are recorded by handleCheckBundle
         // Refreshes the checkbox after commands
         // If the action failed, the check will persist
+        const cmdMsgMap: { string: string } = { rm: ['deleting', 'deleted'] };
         let force_delete = cmd === 'rm' && this.state.forceDelete ? '--force' : null;
         this.setState({ updating: true });
+        let toastMsg =
+            (cmd in cmdMsgMap
```

Comment:
Since we repeat most of this line later on (on line 331), can we perhaps refactor this into a function? This would also make the code a bit clearer to follow.

Something like this:

`getToastMsg(command, state)`

```
getToastMsg("rm", 0) -> "X bundles deleting"
getToastMsg("rm", 1) -> "X bundles deleted"
getToastMsg("kill", 0) -> "Executing kill command"
getToastMsg("kill", 1) -> "Executed kill command"
```

Reply:
Thanks for the advice! Function added.